client/goal-view-ui/src/utilities/pp-break.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {FunctionComponent} from 'react';
+import {PpMode} from '../types';
+
+type PpBreakProps = {
+    id: string,
+    offset: number,
+    mode: PpMode,
+    horizontalIndent: number, 
+    indent: number,
+    lineBreak: boolean,
+};
+
+const ppBreak: FunctionComponent<PpBreakProps> = (props) => {
+    
+    const {mode, lineBreak, indent, horizontalIndent, id, offset} = props;
+    const style = {
+        marginLeft: offset
+    };
+
+    switch(mode) {
+        case PpMode.horizontal:
+            return <span id={id}>{" ".repeat(horizontalIndent)}</span>;
+        case PpMode.vertical:
+            return (
+                <span id={id}>
+                    <br/>
+                    <span style={style}>
+                        {" ".repeat(indent ? indent : 0)}
+                    </span>
+                </span>
+            );
+        case PpMode.hvBox:
+            if(lineBreak) {
+                <span id={id}>
```

Comment:
missing `return` here?

Reply:
Good catch ! I think handling hvBoxes correctly stilll needs a bit of work anyways !